R/anovarepeatedmeasures.R

```diff
@@ -1587,7 +1587,7 @@ AnovaRepeatedMeasuresInternal <- function(jaspResults, dataset = NULL, options)
 
     conoverTable <- createConoverTable(groupingVar)
     noteBlockName <- ifelse(blockingVar == .BANOVAsubjectName, "subject", .BANOVAsubjectName)
```

Comment:
I still got the same bug when no grouping variable was selected. I think this fix should do the trick:

```suggestion
    noteBlockName <- ifelse(blockingVar == .BANOVAsubjectName, "subject", blockingVar)
    conoverTable$addFootnote(gettextf("Grouped by %s.", noteBlockName))
```